scripts/check_frontend_coverage.py

```diff
@@ -468,13 +468,12 @@
     'StoryContentsObjectFactory.ts',
     'StoryNodeObjectFactory.ts',
     'student.ts',
-    'subtopic-editor-modal.controller.ts',
+    'subtopic-preview-tab.directive.ts',
```

Comment:
is this a new directive? If yes, do we need to add test for the same?

Reply:
Yes it is, updated that to a component, yes we should add the test, I intend to do that in a follow-up PR(after probably 2 days). Refer [here](https://github.com/oppia/oppia/pull/9603#discussion_r444711093)